src/lightning_app/cli/lightning_cli.py

```diff
@@ -26,13 +34,149 @@ def get_app_url(runtime_type: RuntimeType, *args) -> str:
         return "http://127.0.0.1:7501/admin" if LOCAL_LAUNCH_ADMIN_VIEW else "http://127.0.0.1:7501/view"
 
 
+@click.group()
+def clusters():
+    """Manage your Lightning.ai compute clusters.
+
+    Lightning.ai supports running apps on your own cloud provider account. This feature is currently in preview and
+    requires an extra activation step. Reach out to support@lightning.ai to enable this feature.
+    """
+    pass
+
+
+@clusters.command("create")
+@click.argument("cluster_name", callback=_check_cluster_name_is_valid)
+@click.option("--provider", "provider", type=str, default="aws", help="cloud provider to be used for your cluster")
+@click.option("--external-id", "external_id", type=str, required=True)
+@click.option(
+    "--role-arn", "role_arn", type=str, required=True, help="AWS role ARN attached to`the associated resources."
+)
+@click.option(
+    "--region",
+    "region",
+    type=str,
+    required=False,
+    default="us-east-1",
+    help="AWS region which is used to host the associated resources.",
```

Comment:
Please use: 

AWS region that is used to host the associated resources.

Reply:
:+1: